docs/remove_rhts.rst

```diff
@@ -0,0 +1,278 @@
+Guide to removing RHTS from Jobs
+================================
+
+For some products, Test Requirements include running restraint by itself.
+This requires the exclusion of the legacy RHTS package or `restraint-rhts`
+package installation.  Below lists areas to draw attention in order
+to eliminate RHTS references.
+
+#. Define only `restraint` and not `restraint-rhts` as your harness in your job's
+   recipe definition.  The recipe line below illustrates this harness setting::
+
+     <recipe whiteboard="" role="RECIPE_MEMBERS" **ks_meta="harness='restraint'"**
+         kernel_options="" kernel_options_post="">
+
+#. Avoid defining tasks or dependencies which cause installation of the `RHTS` library.
+#. Replace `RHTS` scripts with `Restraint` scripts.  :ref:`legacy_rhts_cmds` provides
+   a table which maps legacy to restraint scripts.
+#. Change your tasks to utilize `Restraint's metadata` file instead of `RHTS`
+    `testinfo.desc` file. :ref:`legacy_metadata` provides details on mapping
+   legacy `testinfo.desc` variables to restraint `metadata` variables. Depending
+   on how your task is written, you may have to update or remove `Makefiles` so they
+   do not process the `testinfo.desc` file.  An example of this is also
+   included in the referenced section.
+#. Replace `RHTS` environment variables with `Restraint` variables. A table listing
+   `RHTS Legacy Variables` to `Restraint Substitute` can be found in
+   :ref:`legacy_env_var`.
+
+.. _legacy_rhts_cmds:
+
+Replacement for RHTS Scripts
+----------------------------
+
+The table below lists known legacy RHTS commands.  Some are provided in the
+`restraint-rhts` package and some are from `rhts` package.  It is encouraged for people
+to use `Restraint`'s substitute for these commands as they are actively supported.
+Included in the table are the `Restraint` substitutes and which RHTS commands are deprecated.
+
++--------------------------------+-------------------------------------------+------------+
+| RHTS Legacy Script             | Restraint Substitute                      | Deprecated |
```

Comment:
Am I the only one who is confused by this table? Deprecated is tight to what? RHTS or Restraint here?

Reply:
It seems based on the logic below -> if we have Restraint command it is not deprecated

Reply:
I will remove Deprecated and perhaps it will be clearer.